app/src/main/java/com/jalasoft/routesapp/data/local/room/managers/TourPointLocalManager.kt

```diff
@@ -0,0 +1,14 @@
+package com.jalasoft.routesapp.data.local.room.managers
+
+import com.jalasoft.routesapp.data.local.room.db.RoutesDB
+import com.jalasoft.routesapp.data.local.room.interfaces.TourPointLocalRepository
+import com.jalasoft.routesapp.data.model.remote.TourPointPath
+
+class TourPointLocalManager(private val localRoutesDB: RoutesDB) : TourPointLocalRepository {
+    override fun getAllTourPoints(): List<TourPointPath> {
+        val tourPoints = localRoutesDB.tourPointDao().getAllTourPoints()
```

Comment:
Here is getting all tour points it will include from different cities if were downloaded i think you should get only from the current city selected

Reply:
Fixed, now the tourPoint list are fetched by the current city id.